error: variable 'lua_state_ref' is used uninitialized whenever 'if' condition is true

macOS Catalina, 10.15.6
```
$ make -j httpng
/Users/a.starshov/httpng/httpng/httpng.c:1456:6: error: variable 'lua_state_ref' is used uninitialized whenever 'if' condition is true
      [-Werror,-Wsometimes-uninitialized]
        if (fill_received_headers_and_body(L, shuttle)) {
            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/a.starshov/httpng/httpng/httpng.c:1526:46: note: uninitialized use occurs here
        luaL_unref(luaT_state(), LUA_REGISTRYINDEX, lua_state_ref);
                                                    ^~~~~~~~~~~~~
/Users/a.starshov/httpng/httpng/httpng.c:1456:2: note: remove the 'if' if its condition is always false
        if (fill_received_headers_and_body(L, shuttle)) {
        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/a.starshov/httpng/httpng/httpng.c:1480:2: note: variable 'lua_state_ref' is declared here
        const int lua_state_ref = response->lua_state_ref;
        ^
1 error generated.
make[3]: *** [CMakeFiles/httpng.dir/httpng/httpng.c.o] Error 1
make[2]: *** [CMakeFiles/httpng.dir/all] Error 2
make[1]: *** [CMakeFiles/httpng.dir/rule] Error 2
make: *** [httpng] Error 2
```

--- a/httpng/httpng.c
+++ b/httpng/httpng.c
@@ -1453,6 +1453,7 @@ lua_fiber_func(va_list ap)
 	lua_setfield(L, -2, "method");
 	lua_pushboolean(L, !!response->un.req.ws_client_key_len);
 	lua_setfield(L, -2, "is_websocket");
+	const int lua_state_ref = response->lua_state_ref;
 	if (fill_received_headers_and_body(L, shuttle)) {
 		free_lua_shuttle_from_tx(shuttle);
 		goto Done;
@@ -1477,7 +1478,6 @@ lua_fiber_func(va_list ap)
 	lua_createtable(L, 0, 2);
 	lua_setfield(L, -2, "headers");
 
-	const int lua_state_ref = response->lua_state_ref;
 	if (lua_pcall(L, 2, 1, 0) != LUA_OK) {
 		/* FIXME: Should probably log this instead(?) */
 		fprintf(stderr, "User handler for \"\%s\" failed with error "

